docs/application/dotnet/guides/nui/linear-layout.md

```diff
@@ -0,0 +1,140 @@
+# Linear Layout
+
+[Overview](#overview)<br>
+[Orientation](#orientation)<br>
+[Alignment](#alignment)<br>
+[CellPadding](#cellpadding)<br>
+[Weight](#weight)<br>
```

Comment:
docs.tizen.org automatically generates TOC on the right side. 
Please remove TOC.

Reply:
Okay, Removed TOC.